ui/src/main/java/edu/wpi/grip/ui/pipeline/input/InputSocketController.java

```diff
@@ -59,6 +62,11 @@ protected void initialize() {
     this.type.setText(this.socket.getSocketHint().getTypeLabel());
     this.handle = socketHandleViewFactory.create(this.socket);
     root.add(this.handle, 0, 0);
+    handle.connectedProperty().addListener((observable, oldValue, newValue) -> {
+      if (getContent() != null) {
+        getContent().setDisable(newValue);
```

Comment:
I thought we already did this? We should add a UI test that checks for when a connection is added you can't manually change the UI.